definitions/mobile.d.ts

```diff
@@ -85,8 +85,8 @@ declare module Mobile {
 	}
 
 	interface IAndroidDebugBridge {
-		executeCommand(...args: string[]): IFuture<any>;
-		executeShellCommand(...args: string[]): IFuture<any>;
+		executeCommand(args: string[]): IFuture<any>;
+		executeShellCommand(args: string[]): IFuture<any>;
```

Comment:
Yes, we use the same version of the TypeScript compiler. The problem actually comes after this change https://github.com/telerik/mobile-cli-lib/commit/aa5e00d3e8c0113837aa8828a746f3d0ea2446bb#diff-416bc024035578187882887a05fe7bfaR22. We've replaced `exec` with `spawnFromEvent`.